classes/helper/Helper.php

```diff
@@ -279,29 +253,6 @@ public function renderCategoryTree(
         return $html;
     }
 
-    /**
-     * use translations files to replace english expression.
-     *
-     * @deprecated use Context::getContext()->getTranslator()->trans($id, $parameters, $domain, $locale); instead
```

Comment:
@PrestaEdit I'm not sure about this one to be honest :-)

Reply:
Deprecated since 1.7.0.0

https://github.com/PrestaShop/PrestaShop/pull/5711